python/sidecar/http.py

```diff
@@ -334,3 +328,4 @@ async def get_settings() -> FlatSettingsSchema:
 async def update_settings(req: FlatSettingsSchemaPatch) -> FlatSettingsSchema:
     user_id = "user1"
     return settings.update_settings_for_user(user_id, req)
+    return settings.update_settings_for_user(user_id, req)
```

Comment:
This shouldn't have been merged

Reply:
Nice catch 😅

Reply:
I've noticed this happen on a few of our Python files. I think something that runs when we save Python files must be doing it. Total shot in the dark but maybe it has something to do with missing trailing newlines?